src/org/thoughtcrime/securesms/ApplicationPreferencesActivity.java

```diff
@@ -144,6 +144,8 @@ public void showBackupProvider() {
     Intent intent = new Intent(this, BackupTransferActivity.class);
     intent.putExtra(BackupTransferActivity.TRANSFER_MODE, BackupTransferActivity.TransferMode.SENDER_SHOW_QR.getInt());
     startActivity(intent);
+    overridePendingTransition(0, 0);
```

Comment:
this is to make the activity appear in the same way as the other pages (which are mostly fragments)